docs/sources/setup/install/logging-guide.md

```diff
@@ -0,0 +1,344 @@
+---
+title: Grafana Log Dashboard Guide
+menuTitle: Grafana Log Dashboard Guide
+description: Step-by-step guide for setting up Grafana, Loki, and Grafana Alloy for log aggregation and visualization.
+weight: 310
+---
+
+This guide provides an official, end-to-end reference for deploying a unified log monitoring system using Grafana, Loki, and Grafana Alloy. Centralized log collection is essential for rapid troubleshooting, security auditing, and system performance analysis. By automating both log transfer and ingestion, this setup minimizes operational overhead and ensures that logs from multiple sources arrive in Grafana in near real time.
+
+---
+
+## Step 1: Install Grafana via CLI
+
+Grafana is the visualization layer for metrics and logs. Installing via CLI guarantees reproducible deployments and supports automated provisioning in production environments.  
+* For complete installation instructions on all supported platforms, see the official Grafana guide: [Grafana Installation](https://grafana.com/docs/grafana/latest/setup-grafana/installation/)  
+* This example uses Ubuntu/Debian; if you’re on another OS, adapt the commands below as needed.
+
+```bash
+sudo apt-get update
+sudo apt-get install -y gnupg2 curl
+curl https://packages.grafana.com/gpg.key | sudo apt-key add -
+sudo add-apt-repository "deb https://packages.grafana.com/oss/deb stable main"
+sudo apt-get update
+sudo apt-get install -y grafana
+sudo systemctl start grafana-server
+sudo systemctl enable grafana-server
+````
+
+* Once installed, start Grafana and make sure it's accessible via a port (default: `3000`).
+* After Grafana is running, create an admin account. Once logged in, you'll land on the Grafana homepage.
+
+---
+
+## Step 2: Deploy Loki for Log Storage
+
+Loki provides a scalable and cost-effective approach to storing and querying logs alongside your metrics data. Installing Loki next ensures that you have the backend log store ready for ingestion before configuring collection agents.
+
+* For complete installation instructions on all supported platforms, see the official Loki guide: [Loki Installation](https://grafana.com/docs/loki/latest/setup/install/)
+* This example uses Docker/Docker Compose; if you’re using another approach, adapt the commands below as needed.
```

Comment:
```suggestion
* This example uses Docker/Docker Compose; if you’re using another approach, adapt the examples below as needed.
```

Also, it's helpful to give the name of the yaml file when you provide code examples, since there are so many different files (meaning, don't assume that users know where to copy/paste this).